### DISCUSSION TITLE
Custom auth.user

### DISCUSSION THREAD

**Discussion by shoaibsharif** [adonisjs/core#1662] [Help]
is it possible I can customize `auth.user` when I am getting the logged in user information using edge template

**Comment by DraftProducts** [comment]
Hi, the user object is directly comming from the model provided in the UserProviderContract in the auth.ts config of your project.
The only way is to use a serialize you'r model. 
Here the [documentation page](https://preview.adonisjs.com/guides/models/serializing-models)
